cronjobs/util.py

```diff
@@ -452,6 +452,37 @@ def FTPLogin(ftp_host, ftp_user, ftp_passwd):
     return ftp
 
 
+def FTPChangeDirectory(ftp, remote_dir_path):
```

Comment:
It might have been nice to implement our own FTP class instead of a series of functions...

Reply:
great idea, i'm on it

Reply:
done